packages/manager/modules/pci/src/projects/project/storages/databases/database/pools/pools.controller.js

```diff
@@ -0,0 +1,42 @@
+export default class {
+  /* @ngInject */
+  constructor($translate, CucCloudMessage) {
+    this.$translate = $translate;
+    this.CucCloudMessage = CucCloudMessage;
+  }
+
+  $onInit() {
+    this.initData();
+    this.ADD_POOL_TRACKING = `${this.databasesTrackPrefix}::pools::add_pool_cancel`;
```

Comment:
I think that you should inject the engine name in the tracking. Please see the usage of the `trackDashboard` function